### ISSUE TITLE
Admin Page approval of new users

### ISSUE THREAD

**Issue by 7597208625** [AppGrate/RecyclingSolutions-Issues#5]
IN admin page new user request for each type user is marked under each type of user.

So in New if admin want to decline any user then it says that fill remaining of required data. This case should not be there. 

If admin approve the user then only it should ask that fill up all the required data. If admin decline then there should be warning message that you are declining user registration. If admin say ok then it should be declined. And will go in decline tab.

**Comment by 7597208625** [comment]
Users and companies are not in sync. 

Have approved Anish Polymer in Aggregator but it shows User linked to Anish Polymer as Unverified. 

please check.

**Comment by 7597208625** [comment]
How to verify users ? 

not able to find any option for this.

**Comment by 7597208625** [comment]
In the select state option state should be Alphabetically arranged.

In  addition if we can add option where city name and state automatically came with help of pin code. 

Please check.

**Comment by 7597208625** [comment]
There should be option to upload Pdf (GST Certificate, Cancel Cheque, CCA, CTE, PWM registration etc)

**Comment by 7597208625** [comment]
After creation of new user if we save anything that it should land on PBIOs/Aggregator/Recycler page. 

Please check.

**Comment by prashantgithub82** [comment]
1. IN admin page new user request for each type user is marked under each type of user. So in New if admin want to decline any user then it says that fill remaining of required data. This case should not be there. If admin approve the user then only it should ask that fill up all the required data. If admin decline then there should be warning message that you are declining user registration. If admin say ok then it should be declined. And will go in decline tab. 
1.1. Users and companies are not in sync. Have approved Anish Polymer in Aggregator but it shows User linked to Anish Polymer as Unverified. 
1.2. How to verify users? not able to find any option for this.
**In Progress. Only companies will be approved, corresponding users will be auto approved.**

2. In the select state option state should be Alphabetically arranged. **FIXED**

3. In addition if we can add option where city name and state automatically came with help of pin code. **Will look into this later**

4. There should be option to upload Pdf (GST Certificate, Cancel Cheque, CCA, CTE, PWM registration etc). **Will discuss on it**

5. After creation of new user if we save anything that it should land on PBIOs/Aggregator/Recycler page. **Need more clarification on which page. Do you mean home page?**

**Comment by 7597208625** [comment]
IN admin page new user request for each type user is marked under each type of user. So in New if admin want to decline any user then it says that fill remaining of required data. This case should not be there. If admin approve the user then only it should ask that fill up all the required data. If admin decline then there should be warning message that you are declining user registration. If admin say ok then it should be declined. And will go in decline tab.

**This issue is different. If admin wants to decline any request for new approval then also it says that fill all the mandatory filled to admin also.**  This case should not be there. If admin want to decline any request for new company approval then it should show a warning message that you are rejecting application for approval. On approval of admin in warning message it should be rejected and it would go in declined application. If admin want to delete then there should be option delete record.

**Comment by 7597208625** [comment]
Point no. 5 
Not to home page.
I mean to say it should redirect to last page. 

For example if we approve any aggregator then it after save it should be redirect to Aggregator page where all the aggregator are listed. I hope you understood.

**Comment by prashantgithub82** [comment]
IN admin page new user request for each type user is marked under each type of user. So in New if admin want to decline any user then it says that fill remaining of required data. This case should not be there. If admin approve the user then only it should ask that fill up all the required data. If admin decline then there should be warning message that you are declining user registration. If admin say ok then it should be declined. And will go in decline tab.
This issue is different. If admin wants to decline any request for new approval then also it says that fill all the mandatory filled to admin also. This case should not be there. If admin want to decline any request for new company approval then it should show a warning message that you are rejecting application for approval. On approval of admin in warning message it should be rejected and it would go in declined application. If admin want to delete then there should be option delete record.
**Actually this is happening due data issue. We did change into address fields (added state, city, zipcode etc), so those values are actually not there for this record. We have corrected this and it should be working as expected now.**

Point no. 5. Not to home page. I mean to say it should redirect to last page. For example if we approve any aggregator then it after save it should be redirect to Aggregator page where all the aggregator are listed. I hope you understood.
**In Progress**
